### PR TITLE
Further suggestions

### PR DESCRIPTION
**Pull request by sofstam**
First draft of adding further suggestions to the text.

### CODE REVIEW

File: taxprofiler-manuscript.qmd [hunk 1]
```diff
@@ -134,34 +134,34 @@ affiliations:
...
+A second challenge in taxonomic profiling <!-- Shall we include classification here as well--> is the parameter and database choice. As with tools, there is no one set 'gold standard' database for taxonomic profiling. Different questions and contexts may require different databases, such as when a researcher wants to search for both bacteria_and viruses species in samples, and as an extension of this, taxonomic classifiers may need different settings for each database. Furthermore, as genomic sequencing becomes cheaper and more efficient, the number of publicly available reference genomes are rapidly increasing [@Nasko2018-cl], making the size of databases taxonomic classifiers also much a larger and often outpacing the computational capacity available to researchers. In fact, while this was one of the main motivations behind classifiers such as Kraken2 [@Wood2019-mf], these algorithmic techniques are already becoming insufficient [@Wright2023-yr].
```
Comment (jfy133) [inline taxprofiler-manuscript.qmd:156]: For the comment: We have a sentence above saying we use it interchangeably, but I'll make it consistent across the paragraph at least

File: taxprofiler-manuscript.qmd [hunk 2]
```diff
@@ -189,15 +189,16 @@ All nf-core pipelines are strictly versioned (specified with the Nextflow `-r` f
...
+Removing host DNA is another common preprocessing step in metagenomic studies. This can help speed up run-time, particularly in microbiome studies, where detection of microbes are of interest. Furthermore,
+host-contamination of reference genomes in public databases is common [CITE] and therefore the removal of such sequences can also decrease the risk of false positive taxonomic assignment. To remove multiple hosts or other sequences, all reference genomes can be combined into a single FASTA reference file. Short-read host removal can be carried out with Bowtie2 [@Langmead2012-ik;@Langmead2019-ej] and minimap2 [@Li2018-qq] for long-reads, both in combination with SAMtools [@Li2009-wy;@Danecek2021-gj]. <!-- Shall we write more about the step here? -->
```
Comment (jfy133) [inline taxprofiler-manuscript.qmd:197]: I've been a bit more explicit, see the PR

**Comment by jfy133** [comment]
Comments otherwise addressed here: https://github.com/jfy133/taxprofiler-manuscript/pull/5

**Comment by sofstam** [comment]
This PR is still a draft. You can wait for all my comments, otherwise happy to already review your PR :)

**Comment by jfy133** [comment]
Ah sorry! As you had already left comment on the funding section I thought you had got through it :sweat_smile: 

No worries, keep going and I can keep adding to the other PR :)

**Comment by sofstam** [comment]
Great! Sorry, I just started to push my comments per section 😄

**Comment by sofstam** [comment]
Here come my first comments :)

File: taxprofiler-manuscript.qmd [hunk 3]
```diff
@@ -222,25 +223,31 @@ As of version 1.1.0, the following classifiers and profilers are available: Krak
 | Nucleotide             | alignment based | marker-gene    | MetaPhlAn4 |
 | Nucleotide             | alignment based | marker-gene    | mOTUS      |
 
-: List of nf-core/taxprofiler supported taxonomic profilers as of version 1.1 and their method and support input database types. Sequencing matching type refers to which 'alphabet' is primarily used for matching between a query (read) and a reference (genome/gene). Method refers to the algorithm type used for matching. Reference type refers to the typical sequence type used in database construction of the tool. {#tbl-tool-summaries}
+: List of nf-core/taxprofiler supported taxonomic profilers as of version 1.1.0 and their method and support input database types. Sequencing matching type refers to which 'alphabet' is primarily used for matching between a query (read) and a reference (genome/gene). <!--I do not understand what you mean with 'alphabet' here --> Method refers to the algorithm type used for matching. Reference type refers to the typical sequence type used in database construction of the tool. {#tbl-tool-summaries}
```
Comment (jfy133) [inline taxprofiler-manuscript.qmd:226]:
```suggestion
: List of nf-core/taxprofiler supported taxonomic profilers as of version 1.1.0 and their method and support input database types. Sequencing matching type refers to which 'alphabet' is primarily used for matching between a query (read) and a reference (genome/gene). <!--I do not understand what you mean with 'alphabet' here --> Method refers to the algorithm type used for matching. Reference type refers to the typical sequence type used in database construction of the tool. {#tbl-tool-summaries}
```

For the comment: I used it based on this definitionhttps://web.mit.edu/meme_v4.11.4/share/doc/alphabets.html#:~:text=The%20protein%20alphabet%20contains%20twenty,Z'%20and%20'J'. I've updated to say 'molecular alphabet'

File: taxprofiler-manuscript.qmd [hunk 4]
```diff
@@ -254,17 +261,17 @@ Run metric can be seen in Figure XXXXXX .
...
+Finally, once an overall taxonomic profile is generated, researchers often wish to validate hits through more sensitive and accurate methods such as with read-mapping alignment. The pipeline could be easily extended to have a validation step where, utilising Nextflow's execution parallelism, the input sequences could be aligned back to the reference genomes of only those species with hits from the taxonomic classification. This step could also be particularly useful for researchers in palaeogenomics, where alignment information can be used to authenticate ancient DNA within their samples. <!-- Maybe extend also here for clinical metagenomics? I can get back with a suggestion-->
```
Comment (jfy133) [inline taxprofiler-manuscript.qmd:270]: Yes please!

**Review by jfy133 — COMMENTED** [review]
Repsonses to questions below

**Comment by jfy133** [comment]
Will close this as I've addressed the comments, and you'veopened a second PR but feel free to check again I've incorporated everything if you've not already

**Pull request closed without merging**